tests/test_aquacomputer.py

```diff
@@ -154,12 +184,32 @@ def test_d5next_get_status_from_hwmon(mockD5NextDevice, tmp_path):
     assert sorted(got) == sorted(expected)
 
 
-def test_d5next_set_fixed_speeds_not_supported(mockD5NextDevice):
-    with pytest.raises(NotSupportedByDriver):
-        mockD5NextDevice.set_fixed_speed("fan", 42)
+def test_d5next_set_fixed_speeds_directly(mockD5NextDevice):
```

Comment:
Similarly to `test_d5next_get_status_directly`, this should also test whether `--direct-access` is honored, even if a (complete) hwmon driver is present.

But I do see how this might be getting a bit repetitive/annoying.